internal/diskutil.go

```diff
@@ -171,12 +173,17 @@ func (b BlockDevice) GetPathByID() (string, error) {
 		}
 	}
 	b.PathByID = ""
-	diskByIDDir := filepath.Join(DiskByIDDir, "/*")
-	paths, err := FilePathGlob(diskByIDDir)
-	if err != nil {
-		return "", fmt.Errorf("could not list files in %q: %w", DiskByIDDir, err)
+	allDisks := []string{}
+	// We are going to prefer wwn, scsi and then nvme- disk-ids in that order
+	for _, pattern := range preferredPatterns {
+		disks, err := fileGlobFunc(pattern, DiskByIDDir)
```

Comment:
THB, I don't like calling `FilePathGlob` that often.

I would personally get all by-id disks as today + sort them by preferred prefix (i.e. `/dev/disk/by-id/wwn-` would be before `/dev/disk/by-id/google-`) and _then_ filter them below using `PathEvalsToDiskLabel`. What it wwn-* symlink does not lead to `/dev/$KNAME`?

Reply:
I benchmarked that option - https://gist.github.com/gnufied/47f7d580d953b98178772eddc0cbfcb5 . Using glob is faster than sorting and filtering. See - https://gist.github.com/gnufied/47f7d580d953b98178772eddc0cbfcb5#file-main-go-L24

Reply:
yeah if wwn- doesn't lead to /dev/$KNAME, then we will have to move to scsi- and then to nvme- and finally to other names.  Fastest option is clearly using the trie.

Reply:
Sorry I ended up using grouped by prefix approach in the end.  The bottleneck turned out to be regex matching and using a simpler `strings.HasPrefix` results in much faster code.